jdbc/core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/driver/ShardingSphereDriverURL.java

```diff
@@ -72,4 +72,30 @@ public byte[] toConfigurationBytes() {
             return builder.toString().getBytes(StandardCharsets.UTF_8);
         }
     }
+    
+    private InputStream getResourceAsStream(final String resource) throws IOException {
+        ClassLoader[] classLoader = new ClassLoader[]{
+                Thread.currentThread().getContextClassLoader(),
+                getClass().getClassLoader(),
+                ClassLoader.getSystemClassLoader(),
+        };
```

Comment:
It's better to put them in one line for shorter code

Reply:
`classLoader` could be `classLoaders`

Reply:
Did you test which class loader satisify most cases or the most common case?

Could we put `getClass().getClassLoader()` at first?

Reply:
OK